__test__/FacetedSearchContainer.test.js

```diff
@@ -68,4 +69,13 @@ describe(`FacetedSearchContainer`, () => {
     const tree = renderer.create(<FacetedSearchContainer {...props} />).toJSON()
     expect(tree).toMatchSnapshot()
   })
+
+  test(`selectedSpecies is passed`, () =>{
```

Comment:
The new behaviour you’ve added is to hide the species facet if all the results have the same value. You should change the test to reflect that.